packages/genesys-spark-tokens/snapshot/css/gse-ui-feedback.css

```diff
@@ -91,8 +91,8 @@
   --gse-ui-popover-anchor-height: 13px;
   --gse-ui-progressAndLoading-spinner-foreground: #4856cb;
   --gse-ui-progressAndLoading-spinner-base: #ced2fc;
-  --gse-ui-progressAndLoading-spinner-large: 48px;
-  --gse-ui-progressAndLoading-spinner-small: 16px;
+  --gse-ui-progressAndLoading-spinner-large: 48;
+  --gse-ui-progressAndLoading-spinner-small: 16;
```

Comment:
These are used so the call sight will have to be updated
`width: var(--gse-ui-progressAndLoading-spinner-small);` -> `width: calc(var(--gse-ui-progressAndLoading-spinner-small) * 1px);` for example.

Reply:
@jason-evans-genesys I think you requested this change, yeah?